eng/WpfArcadeSdk/tools/SdkReferences.targets

```diff
@@ -210,4 +125,33 @@
       </Reference>
     </ItemGroup>
   </Target>
+
+  <ItemGroup>
+    <!-- We always want to use the local WindowsBase, not the one coming from the .NET runtime. -->
+    <DefaultReferenceExclusion Include="WindowsBase"/>
+  </ItemGroup>
+
+  <!--
+    Allows excluding specific .NET runtime libraries from being implicitly referenced.
+    For example, to exclude WindowsBase from being implicitly referenced, add the following to your project file:
+
+      <ItemGroup>
+        <DefaultReferenceExclusion Include="WindowsBase" />
+      </ItemGroup>
+  -->
+  <Target Name="FilterImplicitAssemblyReferences"
+          Condition="'@(DefaultReferenceExclusion)' != ''"
+          DependsOnTargets="ResolveProjectReferences"
+          AfterTargets="ResolveTargetingPackAssets">
+    <ItemGroup>
+      <_referenceExclusion Include="@(DefaultReferenceExclusion)">
+        <AssemblyName>%(DefaultReferenceExclusion.Identity)</AssemblyName>
+        <FrameworkReferenceName>Microsoft.NETCore.App</FrameworkReferenceName>
+        <ExternallyResolved>true</ExternallyResolved>
+      </_referenceExclusion>
+
+      <Reference Remove="@(_referenceExclusion)"
+                 MatchOnMetadata="AssemblyName;FrameworkReferenceName;ExternallyResolved"/>
```

Comment:
Did you need all of this - I guess you did this so you wouldn't need to intersect `DefaultReferenceExclusion` with `Reference`

Reply:
I tried a number of different ways, and this seemed to be the cleanest and easiest to maintain.